statistician/tests/init_db.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash      
+
+# index data for integration test
+datacube system init
+datacube system check
+
+datacube product add https://raw.githubusercontent.com/digitalearthafrica/config/master/products/esa_s2_l2a.odc-product.yaml
```

Comment:
lol? assume this is copied from ows? i can help to do a dump.sql so we don't need to index everytime

Reply:
The test will be an on-going process. New test/data could be added in the future. I'd prefer to simply indexing more when it requires other than having to dump a database each time. The indexing takes fractional time compared with the whole test.